go/stellar/stellar.go

```diff
@@ -330,6 +334,25 @@ func SendPayment(ctx context.Context, g *libkb.GlobalContext, remoter remote.Rem
 	return remoter.SubmitTransaction(ctx, payload)
 }
 
+func isAccountFunded(ctx context.Context, remoter remote.Remoter, accountID stellar1.AccountID) (funded bool, err error) {
+	balances, err := remoter.Balances(ctx, accountID)
+	if err != nil {
+		return false, err
+	}
+	for _, b := range balances {
+		if b.Asset.IsNativeXLM() {
+			a, err := amount.ParseInt64(b.Amount)
+			if err != nil {
+				return false, err
+			}
+			if a > 0 {
+				return true, nil
+			}
+		}
+	}
```

Comment:
Is there a case where BalanceXLM doesn't work?  i.e. remoter.Balances is returning more than one native balance?

Reply:
`remoter.Balances` should return `balances = nil` for a non-funded account. And a list including 1 row with non-0 XLM balance for a funded account. If remoter.Balances returned more than one native balance, as long as one of them was non-zero, this code would return true. Is there another case that would trip this?

Reply:
Sorry, I meant why not just use BalanceXLM?

Reply:
Oh! `BalanceXLM` returns an error when not funded. But I realize now this was the only call of `BalanceXLM` so I'll just get rid of it